src/styles/content/tables/table-cell-inactive/styles.less

```diff
@@ -0,0 +1,131 @@
+& when (@table-cell-inactive-enabled) {
+  /* Table */
+
+  .table {
+
+    tr.inactive td,
+    td.inactive {
+      .element-text-style(table-cell-inactive);
```

Comment:
I'm with @jfurrow and would vote against using those mixins due to the named reasons. The CSS that we write should be in line with our needs not those of CNVS.